packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -92,10 +92,15 @@ const generateCells = (
 ) => {
   if (schema.type === 'object') {
     return getValidColumnProps(schema).map(prop => {
+      let title = startCase(prop);
+      if (schema.properties && schema.properties[prop].title) {
+        title = schema.properties[prop].title;
+      }
```

Comment:
Can be simplified to
```suggestion
      const title = schema.properties?.[prop]?.title ?? startCase(prop);
```